### ISSUE TITLE
no grave when dying to rotarycraft grinder

### ISSUE THREAD

**Issue by Turmfalke2** [OpenMods/OpenBlocks#738]
hi,
I noticed that if bump into the rotarycraft grinder and die from the damage it deals, openblocks fails to place a grave and the items just disappear.

log: http://fpaste.org/305724/

RotaryCraft 1.7.10 V9
DragonAPI 1.7.10 V9b
OpenBlocks-1.7.10-1.4.4

**Comment by boq** [comment]
Thank you for report with logs!
Unfortunately for you, everything is working exactly as expected:

```
[18:06:15] [Server thread/WARN] [OpenMods/]: openblocks.common.PlayerDeathHandler.onPlayerDrops(PlayerDeathHandler.java:332): Event for player 'EntityPlayerMP['Turmfalke_'/859, l='world #1', x=435.06, y=69.88, z=212.20]' cancelled, grave will not be spawned
```

Event is cancelled by one of the handlers. Specifically this one:

```
Reika.RotaryCraft.RotaryEventManager@29c22668 meatGrinding
```

This is the code. I think author intentions are clear.
https://github.com/ReikaKalseki/RotaryCraft/blob/master/RotaryEventManager.java#L319

```java
public void meatGrinding(LivingDropsEvent ev) {
		if (ev.source instanceof GrinderDamage) {
			ItemStack food = ReikaEntityHelper.getFoodItem(ev.entityLiving);
			ev.drops.clear();
			if (food != null) {
				...
			}
			ev.setCanceled(true);
		}
	}
```
**TL;DR;** RotaryCraft explicitly removes items, no recovery possible.
